tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -461,4 +513,164 @@ Check amended release doesn't contain deleted subject
\ No newline at end of file
+    user checks page does not contain  ${SECOND_SUBJECT}
+
+Return to admin to modify footnotes
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+    user waits until h1 is visible   Dashboard
+    user waits until page contains title caption  Welcome Bau1
+
+Create amendment to add and modify footnotes
+    [Tags]  HappyPath
+    user clicks link  Home
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${RELEASE_NAME} (Live - Latest release)  ${accordion}
+    ${details}=  user gets details content element  ${RELEASE_NAME} (Live - Latest release)  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+Add "upload file test filter" subject file
+    [Tags]  HappyPath
+    user clicks link  Data and files
+     user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user enters text into element  id:dataFileUploadForm-subjectTitle   New subject file footnotes
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}upload-file-test-with-filter.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}upload-file-test-with-filter.meta.csv
+    user clicks button  Upload data files
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   New subject file footnotes
+    user opens accordion section   New subject file footnotes
+    ${section}=  user gets accordion section content element  New subject file footnotes
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+Add meta guidance to "New subject file" subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  New subject file footnotes
+    user enters text into meta guidance data file content editor  New subject file footnotes
+    ...  meta guidance content
+    user clicks button  Save guidance
+
+Navigate to 'Footnotes' Tab
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add footnote to "upload file test filter" subject file
+    [Tags]  HappyPath
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote checkbox  Select all indicators and filters for this subject
```

Comment:
This test is brittle as it will always select the first `Select all indicators and filters for this subject` checkbox. If someone changes the tests such that the `Upload file test with filter`/`New subject file footnotes` subject isn't first in the footnotes list, this test will break.

I would first get the `fieldset` element for that particular subject using the `data-testid`, and then click the child element from there. Not sure how other test suites do it - probably be worth looking at how they select a particular subject on the footnotes page and copying that.